mdata/cache/accnt/flat_accnt.go

```diff
@@ -251,12 +284,16 @@ func (a *FlatAccnt) delMet(metric schema.AMKey) {
 	}
 
 	cacheSizeUsed.DecUint64(met.total)
+	cacheOverheadFlat.DecUint64(totalFlat)
+	cacheOverheadLru.DecUint64(totalLru)
+	cacheOverheadChunk.DecUint64(totalChunk)
```

Comment:
that's not important at all: it seems kind of mixed up to me that this function will first delete all the chunks of the metric, then update the related stats about those chunks, and then delete the metric from the metrics map. i think it should either first do all the deletes, and then update the stats, or the other way around.

Reply:
Looking at it now, it does look a little weird. I will change it to update the stats and then delete everything.